UI/data/locale/en-US.ini

```diff
@@ -887,7 +887,7 @@ Basic.Settings.General.MultiviewLayout.25Scene="Scenes only (25 Scenes)"
 Basic.Settings.Stream="Stream"
 Basic.Settings.Stream.StreamType="Stream Type"
 Basic.Settings.Stream.Custom.UseAuthentication="Use authentication"
-Basic.Settings.Stream.Custom.Username="Username"
+Basic.Settings.Stream.Custom.Username="Username /SRT Streamid"
```

Comment:
Should be one more space after `/`, though I think I would prefer keeping this as "Username" and changing the entire string to "Streamid" if the selected protocol is SRT.

Reply:
I can remove streamid.
In terms of UI/UX, it's actually not ideal to have srt/rist in the same service as custom rtmp.
It'd be better to have a separate service.
There's the idea that more specific options could then be offered. I'm thinking of the main ones, latency, bandwidth and profile.
There's also the services/outputs/protocols overhaul which is in RFC (from tytan). So there's a timing issue which makes it unclear for me about the best path forward.
For now, i'll remove the streamid string.
The information will be relegated to the wiki.
For upcoming srt services (this is what prompted this PR), that's not ideal though. They'll have to walkthrough their users as to what is what.

Reply:
removed the streamid string